Dashboard/app/js/lib/analytics.js

```diff
@@ -1,20 +1,41 @@
-const _paq = window._paq || [];
-_paq.push(['setDocumentTitle', `${document.domain}/${document.title}`]);
-_paq.push(['setCookieDomain', '*.akvoflow.org']);
-_paq.push(['setDomains', ['*.akvoflow.org']]);
-_paq.push(['trackPageView']);
-_paq.push(['enableLinkTracking']);
+export function init() {
+  window._paq = window._paq || [];
 
-(function () {
-  const u = `${(document.location.protocol == 'https:') ? 'https' : 'http'}://analytics.akvo.org/`;
-  _paq.push(['setTrackerUrl', `${u}piwik.php`]);
-  _paq.push(['setSiteId', '10']);
-  const d = document;
-  const g = d.createElement('script');
-  const s = d.getElementsByTagName('script')[0];
-  g.type = 'text/javascript';
-  g.defer = true;
-  g.async = true;
-  g.src = `${u}piwik.js`;
-  s.parentNode.insertBefore(g, s);
-}());
+  window._paq.push(['setDocumentTitle', `${document.domain}/${document.title}`]);
+  window._paq.push(['setCookieDomain', '*.akvoflow.org']);
+  window._paq.push(['setDomains', ['*.akvoflow.org']]);
+  window._paq.push(['enableLinkTracking']);
+
+  (function() {
+    const u =
+      document.location.protocol === 'https:'
+        ? 'https://akvo.piwikpro.com/'
+        : 'http://analytics.akvo.org/';
```

Comment:
Let's always use `https`, so `const u = 'https://akvo.piwikpro.com'`.
Protocol relative URL are an anti-pattern now - https://www.paulirish.com/2010/the-protocol-relative-url/